extensions/common/api/api-core/src/main/java/org/eclipse/edc/api/model/IdResponseDto.java

```diff
@@ -18,8 +18,14 @@
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
 
+import static org.eclipse.edc.spi.CoreConstants.EDC_NAMESPACE;
+
 @JsonDeserialize(builder = IdResponseDto.Builder.class)
 public class IdResponseDto extends BaseResponseDto {
+
+    public static final String EDC_ID_RESPONSE_DTO_TYPE = EDC_NAMESPACE + "IdResponseDto";
```

Comment:
nit:  naming this just `TYPE` and `CREATED_AT` would be sufficient and keeps code more readable IMO

Reply:
there would be a clash anyway, so you'd probably have to use `IdResponseDto.TYPE`, no?